### DISCUSSION TITLE
Unable to get into "base" environment

### DISCUSSION THREAD

**Discussion by hardud** [idaholab/moose#26178] [Q&A Getting Started]
I have successfully installed Mambaforge in my home directory and exported the path so it can be used.
When I run "mamba init" it initializes, and asks to close and reopen my current shell. 
When I reopen my shell, I AM NOT GETTING INTO THE BASE ENVIRONMENT. I DO NOT SEE MY PROMPT PREFIXED WITH (BASE).  I am stuck in the loop since forever.

**Comment by GiudGiud** [comment]
Hello

Which os/platform is this?
Open the .bashrc / profile (depending on the platform) and look for the prompt that starts mamba on console startup. is it there?

Guillaume

**Reply by hardud** [reply]
I have macOS.

My .bash_profile reads the following:

<img width="698" alt="Screenshot 2023-11-28 at 9 51 52 PM" src="https://github.com/idaholab/moose/assets/144852214/e757a08e-b35a-4274-abce-55bba51a529e">

**Reply by GiudGiud** [reply]
are you using zsh for the shell by any chance? if so the .bash_profile is completely ignored. 
You ll have to modify the .zshprofile or stop using zsh (and use bash instead)

**Reply by hardud** [reply]
That worked! Thank you so much.

**Reply by milljm** [reply]
Another fix would be to run (something only recently discovered):

```bash
mamba init --all
```

**Reply by GiudGiud** [reply]
that s genius. Making a PR to change our instructions rn
